From 92f2d80e3e8c474efb309cec78ac0cadc1ea0c54 Mon Sep 17 00:00:00 2001
From: Douglas Kosovic <doug@uq.edu.au>
Date: Sat, 20 Mar 2021 12:17:50 +1000
Subject: [PATCH] Remove GPLv2/OpenSSL license conflict in README.md

Debian ftp-master has confirmed the OpenSSL system library exception can
be invoked like Fedora does. OpenSSL, cups, and libgcc are now considered
system libraries:

http://meetbot.debian.net/debian-ftp/2020/debian-ftp.2020-03-13-20.02.html
---
 README.md | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/README.md b/README.md
index 01a7c90..b3ad17b 100644
--- a/README.md
+++ b/README.md
@@ -1,13 +1,5 @@
 # NetworkMananger-l2tp
 
-----
-If you wish to distribute NetworkManager-l2tp 1.8.6 binaries for a Linux
-distribution, please note that there is a GPL/OpenSSL license conflict
-with OpenSSL < 3.0.0 on Linux distibutions that do not consider OpenSSL
-(or LibreSSL) to be a "System Library". See release notes for further details:
-* https://github.com/nm-l2tp/NetworkManager-l2tp/releases
-----
-
 NetworkManager-l2tp is a VPN plugin for NetworkManager 1.8 and later which
 provides support for L2TP and L2TP/IPsec (i.e. L2TP over IPsec) connections.
 
